### PR TITLE
Add Kie server S2I drools integration test

### PR DESCRIPTION
**Pull request by sutaakar**

### CODE REVIEW

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iDroolsIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                .build();
+
+        return deploymentScenarioFactory.getGenericScenarioBuilder()
+                .withKieServer(kieServerS2Isettings)
+                .build();
+    }
+
+    @Before
+    public void setUp() throws ClassNotFoundException {
+        deploymentScenario.getKieServerDeployments().get(0).setRouterTimeout(Duration.ofMinutes(5));
```
Comment (jakubschwan) [inline test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iDroolsIntegrationTest.java]: In this test case should be rule processed immediately, right? So there is no need to set OpenShift router timeout.

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iDroolsIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        Object person = KieServerReflections.createInstance(PERSON_CLASS_NAME, extraClasses.iterator().next().getClassLoader(), PERSON_NAME, "");
+        commands.add(commandsFactory.newInsert(person, PERSON_OUT_IDENTIFIER));
+
+        ServiceResponse<ExecutionResults> reply = ruleClient.executeCommandsWithResults(CONTAINER_ID, executionCommand);
+        assertEquals(ServiceResponse.ResponseType.SUCCESS, reply.getType());
+
+        ExecutionResults actualData = reply.getResult();
+        Object value = actualData.getValue(PERSON_OUT_IDENTIFIER);
+
+        assertEquals("Expected surname to be set to 'Vader'",
```
Comment (jakubschwan) [inline test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iDroolsIntegrationTest.java]: Can you replace this junit Assert with assertj?

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iDroolsIntegrationTest.java (hunk 1 above)
Comment (sutaakar) [reply test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iDroolsIntegrationTest.java]: Right, it was a leftover, good catch, fixed.

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iDroolsIntegrationTest.java (hunk 2 above)
Comment (sutaakar) [reply test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iDroolsIntegrationTest.java]: Done.

**Pull request merged**